components/ToDoForm.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components";
+import Image from "next/image";
+import addButtonCircleActive from "../assets/addButtonCircleActive.svg";
+
+export default function ToDoForm({ onSubmitNewToDoItem, destinationId }) {
+  return (
+    <ToDoListForm
+      onSubmit={(event) => onSubmitNewToDoItem(event, destinationId)}
+      aria-label="add todo form"
+    >
+      <AddToDoButton aria-label="submit">
+        <Image src={addButtonCircleActive} width="20x" height="20px" alt="Add icon" />
+      </AddToDoButton>
+      <ToDoInput
+        name="todo"
+        aria-label="todo"
+        placeholder="Add To-Do..."
+        autoComplete="off"
+        required
+      />
+    </ToDoListForm>
+  );
+}
+
+const ToDoInput = styled.input`
+  margin-left: 0.8em;
+  width: 90%;
+  height: 1.8em;
+  filter: drop-shadow(0 0 0.3em var(--drop-shadow));
+  transition: all ease-in-out 200ms;
+  border: 0;
+  border-radius: 50px;
```

Comment:
margin, padding and height scale with em. You probably want the border-radius to also scale with em. If you change the font-size of this element it will not scale proportionally otherwise.